SConstruct

```diff
@@ -17,6 +17,8 @@ if platform.system() == "Darwin":
 if arch == "aarch64" and not os.path.isdir("/system"):
   arch = "larch64"
 
+is_ubuntu = 'ubuntu' in os.getenv('DESKTOP_SESSION', 'unknown')
```

Comment:
This isn't a reliable way to detect the OS, returns `i3` (window manager) on my machine